[2024-04-13 16:49:00] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:49:05] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:49:10] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:49:15] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:49:17] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 6c72c34988764114 [::1]:37228 failed checkpoint posthandshake: Connected to self
[2024-04-13 16:49:17] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn static dial conn 6c72c34988764114 [::1]:20131 failed checkpoint posthandshake: Connected to self
[2024-04-13 16:49:20] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:49:25] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:49:30] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:49:35] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:49:40] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:49:45] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:49:47] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 6c72c34988764114 [::1]:39384 failed checkpoint posthandshake: Connected to self
[2024-04-13 16:49:47] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn static dial conn 6c72c34988764114 [::1]:20131 failed checkpoint posthandshake: Connected to self
[2024-04-13 16:49:50] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:49:55] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
